.pre-commit-config.yaml

```diff
@@ -7,3 +7,7 @@ repos:
     entry: Unstage generated files with 'git reset --' command
     files: '^(src/main|api_docs/v)'
     verbose: true
+  - id: java-format
+    name: Format code
+    language: script
+    entry: ./format.sh
```

Comment:
can you use https://pre-commit.com/#docker definition for pre-commit hoot?

Reply:
I think locally you can only use `docker-image` https://pre-commit.com/#repository-local-hooks
And we'll change this approach completely.